app/src/main/res/layout/view_background_message.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        tools:src="@drawable/elephant_offline" />
```

Comment:
I would make the image a drawableTop of the textview (since it is only decoration and is not important for accessibility)

Reply:
That's a very good suggestion, now I understand why it was done this way!
There's a catch though: it means that it won't be dynamically resizable (I was not sure if it's needed or not and for not it's not needed).
we could achieve the same by setting description to `@null` I think?
I will try to apply your suggestion anyway.